docs/contents/cloud-dialogflow.json

```diff
@@ -0,0 +1,10 @@
+{
+    "title": "Dialogflow",
+    "services": [
```

Comment:
Hmm, it looks like this should more closely match what we have [here](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/docs/contents/cloud-iot.json). This may be something else we may need to get added to the tooling.